sdk.js

```diff
@@ -355,7 +428,15 @@ class PaddleSDK {
 	 * const result = await client.updateSubscriptionPlan(123, { quantity: 2 });
 	 */
 	updateSubscription(subscriptionID, postData) {
-		const { quantity, price, planID, currency } = postData;
+		const {
+			quantity,
+			price,
+			planID,
+			currency,
+			prorate,
+			keepModifiers,
+			billImmediately,
+		} = postData;
```

Comment:
Add the new attributes in the JSDoc above

Reply:
I tried updating the `yarn.lock` as there were packages with security issues. Is there some way I could update the packages to secure ones in another way then?

Reply:
I understand, but please create a separate PR for that update.